docs/docs/modules/data_connection/document_transformers/semantic_character_text_splitter.ipynb

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
i think combining these into one notebook is probably best, theyre pretty similar

Reply:
> i think combining these into one notebook is probably best, theyre pretty similar

@hwchase17  Ok Yes, Combined these 2 Splitter in one docs